global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.0.100-preview-009764",
+    "dotnet": "3.0.100-preview-010024",
```

Comment:
it looks like this version of dotnet sdk requires v 16.0 of MSBuild but only 15.9.21.664 is available? @ericstj do you know why that would be? and how do i know what version to use that requires a specific version of MSBuild?

Reply:
Is that available on the build machines now?  Or will Arcade downgrade as needed?

Reply:
@livarcocc / @nguerrera  how can folks build using the latest SDK + machine/wide MSBuild?  Are there different machines to use (or some property to suppress this check)?

Reply:
There is a dev16 build machine pool that we can use AFAIK. It may not have the C++ compiler bits we WPF needs etc., but it surely has at least Preview 1. Regardless, WPF should bump vs.version to 16.0 because that's the min we support. This is separate from any other msbuild concerns.

Reply:
Indeed.  So long as you do that and you are able to ensure you're building with the locally acquired SDK (though using the machine-wide MSBuild) you should be back on track for getting updates to NETCore.App you build/test against.